### PR TITLE
Adding production stacks and configuration for edx Maple deployment

### PR DESCRIPTION
**Pull request by blarghmatey**
- Adding SOPS secrets files with values pulled from current production
- Updated QA secrets with values pulled from currently running QA environments
- Added production stacks for edxapp deployment
- Fixed configuration templates to remove unset Consul keys

### CODE REVIEW

File: src/bilder/images/edxapp/templates/edxapp/mitx-staging/common_values.yml [hunk 1]
```diff
@@ -457,7 +457,7 @@ SENTRY_ENVIRONMENT: {{ env "ENVIRONMENT" }}
 SERVER_EMAIL: odl-devops@mit.edu # MODIFIED
 # Removing the session cookie domain as it is no longer needed for sharing the cookie
 # between LMS and Studio (TMM 2021-10-22)
-# SESSION_COOKIE_DOMAIN: {{ key "edxapp/cookie-domain" }} # MODIFIED
+# SESSION_COOKIE_DOMAIN:  # MODIFIED
```
Comment (shaidar) [inline src/bilder/images/edxapp/templates/edxapp/mitx-staging/common_values.yml:460]: Do we need to keep the key if it has no value set?

Comment (blarghmatey) [reply src/bilder/images/edxapp/templates/edxapp/mitx-staging/common_values.yml:460]: Most likely not, but I'm being conservative while we work through this deployment. It also acts as a reminder to dig into it when we have the time to trim down configuration settings even further.

File: src/ol_infrastructure/applications/edxapp/Pulumi.applications.edxapp.mitx-staging.Production.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+config:
+  aws:region: us-east-1
+  consul:address: https://consul-mitx-qa.odl.mit.edu
+  edxapp:business_unit: residential-staging
+  edxapp:db_password:
+    secure: v1:ZWtTTF60+1g61/XW:K9osJstG/oeHJvYXOUQS6KSIUuTcQFgR1cjECmosVk7BPLCJMPkzlLviW7swdiUhUIPErfyYNqs=
+  edxapp:db_storage_gb: '150'
+  edxapp:dns_zone: mitx
+  edxapp:domains:
+    lms: mitx-qa-draft.mitx.mit.edu
```
Comment (shaidar) [inline src/ol_infrastructure/applications/edxapp/Pulumi.applications.edxapp.mitx-staging.Production.yaml:13]: copy pasta?

File: src/ol_infrastructure/applications/edxapp/Pulumi.applications.edxapp.mitx-staging.Production.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+secretsprovider: awskms://alias/infrastructure-secrets-production
+encryptedkey: AQICAHgQalNS7T35ZlcFdhF0QuKeiJAbXMUbm01pjGwHEsjRCgEgvt/Jaq3CayTQkSpHSpvzAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMIIzka63p6WK1zqpCAgEQgDuzHVJL9JoKHJrHL3HMGVlw+oOoUnrWXUMMJfSRRoPkgHnYqhhmrjeIm/OCaK1iFSi27868w83Hwpzj1w==
+config:
+  aws:region: us-east-1
+  consul:address: https://consul-mitx-qa.odl.mit.edu
```
Comment (shaidar) [inline src/ol_infrastructure/applications/edxapp/Pulumi.applications.edxapp.mitx-staging.Production.yaml:6]: copy pasta?

File: src/ol_infrastructure/applications/edxapp/Pulumi.applications.edxapp.xpro.Production.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+secretsprovider: awskms://alias/infrastructure-secrets-production
+encryptedkey: AQICAHgQalNS7T35ZlcFdhF0QuKeiJAbXMUbm01pjGwHEsjRCgF4aGd/aPWQE/Xl6+yg2i3QAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQML9B70uczH/BrDBcvAgEQgDvj4qpjZtDzwWyzwS//9PivKEIscFC5isQm6YJKeBcTP7pxVAxiGSEdeOf/6KANIcQX4RYoroRw6WYMXA==
+config:
+  edxapp:db_password:
+    secure: v1:U83x0i6WAndxuOLK:XsAILALTGbtjCqGJllcqg3EaOM0FF/SNmuiMAK2r3E9X5joW26kjOQzYUcKr0slkHBv8RUEr3xA=
+  edxapp:edx_forum_secrets:
+    secure: v1:YHqAWSdn7TNUn7d+:IBgiGsl4r8vymID8NV43rnfJy+qsJSXm/jXaWWALUfF9zXqRO4zJZhLS0d+etijGVwaZnF8v5zXlo+EczFrCe7ICHpS85zHgifbfDcw=
+  edxapp:google_analytics_id: UA-5145472-38
+  aws:region: us-east-1
+  consul:address: https://consul-xpro-qa.odl.mit.edu
```
Comment (shaidar) [inline src/ol_infrastructure/applications/edxapp/Pulumi.applications.edxapp.xpro.Production.yaml:11]: copy pasta?

**Comment by shaidar** [comment]
👍

**Pull request merged**